mkdocs.yml

```diff
@@ -147,7 +147,9 @@ nav:
         - API server source:
           - Overview: eventing/sources/apiserversource/README.md
           - Getting started: eventing/sources/apiserversource/getting-started/README.md
-        - ContainerSource: eventing/sources/containersource.md
+        - ContainerSource:
+          - Creating a ContainerSource object: eventing/sources/containersource/README.md
+          - ContainerSource Reference: eventing/sources/containersource/reference.md
```

Comment:
While we're moving this I'd go ahead and move it to the developer guide instead?

Reply:
I created an Eventing section in the dev guide and moved the ContainerSource topics over. Let me know if that lines up with what you were thinking.